boa/contracts/vyper/compiler_utils.py

```diff
@@ -157,3 +157,20 @@ def __boa_debug__() {return_sig}:
     """
     )
     return compile_vyper_function(wrapper_code, contract)
+
+
+DEFAULT_BLUEPRINT_PREAMBLE = b"\xFE\x71\x00"
+
+
+def generate_blueprint_bytecode(
```

Comment:
there's not really anything specific about this to vyper, it can go in `base_evm_contract.py`